pkg/core/variationDecider_test.go

```diff
@@ -194,7 +194,7 @@ func TestGetVariationFromUserStorage(t *testing.T) {
 
 	campaign := vwoInstance.SettingsFile.Campaigns[0]
 	userID := "USER_3"
-	actual, err := GetVariationFromUserStorage(vwoInstance, userID, campaign)
+	actual, _, err := GetVariationFromUserStorage(vwoInstance, userID, campaign)
```

Comment:
do not ignore values in the test file. write an assert for the same.

Reply:
Ok